packages/block-editor/src/components/rich-text/test/index.perf-test.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { measurePerformance } from 'reassure';
+import { fireEvent } from '@testing-library/react-native';
```

Comment:
Are we able to leverage `test/helpers` here and the rest of the test files? 
```suggestion
import { fireEvent } from 'test/helpers';
```

Reply:
Updated.